ui/src/components/drafts/components/DepositAccess.js

```diff
@@ -17,9 +17,8 @@ import CheckBox from "grommet/components/CheckBox";
 import RadioButton from "grommet/components/RadioButton";
 import FormField from "grommet/components/FormField";
 import TextInput from "grommet/components/TextInput";
-import cogoToast from "cogo-toast";
 
-import { handlePermissions, clearError } from "../../../actions/draftItem";
```

Comment:
remove `clearError` from actions also..its not used anywhere else (also its a duplicate of `clearErrorSuccess`)

Reply:
```clearError``` has been removed.
We should investigate more on the ```clearErrorSuccess``` since it is imported to the DraftEditor but is never used.